tests/topotests/bgp_srv6l3vpn_to_bgp_vrf2/r2/zebra.conf

```diff
@@ -0,0 +1,41 @@
+log file zebra.log
+!
+hostname r2
+password zebra
+!
+log stdout notifications
+log monitor notifications
+log commands
+!
+debug zebra packet
+debug zebra dplane
+debug zebra kernel
+!
+interface eth0
+ ipv6 address 2001::2/64
+!
+interface eth1 vrf vrf10
+ ip address 192.168.2.1/24
+ ipv6 address 2001:2::1/64
```

Comment:
If you don't support vpnv6 on this PR. this ipv6 addresses is not needed, right?

Reply:
Yes, I forgot to delete it

Reply:
fixed